README.md

```diff
@@ -49,9 +49,6 @@ First you need to install
     [Twilio Account Settings](https://www.twilio.com/console).
```

Comment:
This actually requires `TWILIO_API_KEY` and `TWILIO_API_SECRET` not `TWILIO_AUTH_TOKEN` and it will also require `TWILIO_APP_SID`